graylog2-server/src/main/java/org/graylog2/bootstrap/ServerBootstrap.java

```diff
@@ -208,7 +208,10 @@ private void doRunWithPreflightInjector(Injector preflightInjector) {
 
         final ServiceManager serviceManager = preflightInjector.getInstance(ServiceManager.class);
         final LeaderElectionService leaderElectionService = preflightInjector.getInstance(LeaderElectionService.class);
+        final Service manageableLeaderElectionService = preflightInjector.getInstance(Key.get(Service.class, Names.named("LeaderElectionService")));
```

Comment:
Is there a reason why we can't obtain an instance directly by requesting LeaderElectionService.class ?

Reply:
That's why I asked @thll and @mpfz0r for a review, too. If you look into `LeaderElectionModule` you can see that the binding is a bit different from what we usually do. That's why I'm looking it up twice because I need both interfaces to work and don't just want to typecast. Maybe there is a better way?

Reply:
I don't quite remember why we didn't make the `LeaderElection` interface extend `Service`. I'll look into it first thing next week..

Reply:
Apparently we have introduced the extra binding [here](https://github.com/Graylog2/graylog2-server/pull/11719). But I can't recall why and I guess we can now just omit binding the Service interface and instead have `LeaderElectionService` extend `Service`. I'll create an extra PR to change that,  so that you can merge this one already. Even though it looks a bit ugly now, I don't think there's anything wrong with it.